docfx_yaml/extension.py

```diff
@@ -1951,6 +2033,41 @@ def convert_module_to_package_if_needed(obj):
 
         file_name_set.add(filename)
 
+        for entry in yaml_data:
+            summary_type = _SUMMARY_TYPE_BY_ITEM_TYPE.get(entry.get("type"))
+            if not (summary_type):
+              continue
+
+            _ENTRY_NAME_AND_ENTRY_CONTENT_BY_SUMMARY_TYPE[summary_type][1].append(
+                _find_summary_details(entry, summary_type, cgc_url)
```

Comment:
Nit:  I think because the return value of ` _find_summary_details` is only used to update the global mapping, I would just append the summary detail dict within the function.

Reply:
Done! Good point.